admission_control/admission_control_service/src/admission_control_fuzzing.rs

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use super::AdmissionControlService;
+use crate::{
+    admission_control_service::SubmitTransactionRequest,
+    mocks::local_mock_mempool::LocalMockMempool,
+};
+use proptest;
+use proptest_helpers::ValueGenerator;
+use proto_conv::IntoProto;
+use protobuf;
+use std::sync::Arc;
+use storage_service::mocks::mock_storage_client::MockStorageReadClient;
+use types::transaction::SignedTransaction;
+use vm_validator::mocks::mock_vm_validator::MockVMValidator;
+
+#[test]
+fn test_fuzzer() {
+    let mut gen = ValueGenerator::new();
+    let data = generate_corpus(&mut gen);
+    fuzzer(&data);
+}
+
+/// generate_corpus produces a
+pub fn generate_corpus(gen: &mut ValueGenerator) -> Vec<u8> {
```

Comment:
Can you implement this as a strategy that generates a `SubmitTransactionRequest`?

Reply:
Ah, didn't realize that `SubmitTransactionRequest` is just a protobuf wrapper. This is fine in that case -- though maybe it should live in the fuzzer code directly.

Reply:
it needs to live here as I'm using it for a test : o